frontend/src/app/features/invite-user-modal/project-selection/project-selection.component.ts

```diff
@@ -141,20 +155,21 @@ export class ProjectSelectionComponent implements OnInit {
       return;
     }
 
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
     const projectId = idFromLink(this.projectControl?.value?.href);
     const project = await this.apiV3Service.projects.id(projectId).get().toPromise();
 
     this.save.emit({
       project,
-      type: this.typeControl?.value,
+      type: this.typeControl.value as string,
     });
   }
 
   APIFiltersForProjects = [['active', '=', true]];
 
   projectFilterFn(projects:IProjectAutocompleteItem[]):IProjectAutocompleteItem[] {
     const mapped = projects.map((project) => {
-      const disabled = !this.projectInviteCapabilities.find((cap) => parseInt(cap.context?.id, 10) === project.id);
+      const disabled = !this.projectInviteCapabilities.find((cap) => idFromLink(cap._links.context.href) === project.id.toString());
```

Comment:
We really need to set a standard of whether IDs are strings or numbers in our application. It's a bit of a mess.

Reply:
I agree, and it should be string (as we have non-numerical IDs in some cases such as filter operators)